doc/examples/edges/plot_ridge_filter.py

```diff
@@ -65,7 +65,7 @@ def identity(image, **kwargs):
     for j, func in enumerate([identity, meijering, sato, frangi, hessian]):
         kwargs['black_ridges'] = black_ridges
         result = func(image, **kwargs)
-        if func in (meijering, frangi):
+        if func in (meijering):
```

Comment:
you need to put the comma back `(meijering,)` otherwise it's not iterable. But do we need a loop if there is only one element?

Reply:
see the failing doc build in the CI